src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -116,6 +120,64 @@ object JSONUtil {
   }
 
   def writeToJSON[T <: Node](x: T, os: BufferedWriter): Unit = {
-    os.write(toJSON(x).toString)
+    os.write(toJSON(x))
```

Comment:
`toJSON` is a public method, so seems that this method is not generic enough to be implemented as a separate method.